spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/integration/inbound/PubSubInboundChannelAdapter.java

```diff
@@ -52,6 +53,8 @@
 
 	private MessageConverter messageConverter;
 
+	private HeaderMapper<Map<String, String>> headerMapper = new PubSubHeaderMapper();
```

Comment:
Please, don't forget to update a Copyright in all the affected classes to reflect the current year, e.g. here it must be like `2017-2018`.

Reply:
Changed.

Would be great if this could be automated somehow with IntelliJ...

Reply:
Out of scope of this PR...
We have to follow policy this or other way - nothing personal.